content/java/concepts/math-methods/terms/ulp/ulp.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: '.ulp()'
+Description: 'Returns the size of the unit of last precision of a number.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Documentation'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **Math.ulp()** method returns the unit of least precision of a given number. It calculates the distance between the given `float` or `double` value and the `float` `double` value next larger in magnitude.
+
+## Syntax
+
+```pseudo
+Math.ulp(num);
+```
+
+- The `num` parameter can either be of type `float` or `double`.
+- The return type will be the same as the parameter type.
+
+Special cases include the following:
+
+- If `num` is `NaN`, `NaN` will be returned.
+- If `num` is positive or negative zero, then the return value will be `Double.MAX_VALUE` or `Float.MAX_VALUE` which is the minimum numbers that `float` or `double` can represent.
```

Comment:
The return value for a positive or negative zero is *`Double.MIN_VALUE`* (or *`Float.MIN_VALUE`*), the lowest number a `float` or `double` can represent.

https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#ulp-double-

Reply:
Of course I will fix that asap

Reply:
done now it should be correct! also I used a slightly different phrasing.